pkg/informer/supportedResources.go

```diff
@@ -37,7 +37,7 @@ func GetAllowDenyData(cm *v1.ConfigMap) ([]Resource, []Resource) {
 		Can't use configured value, defaulting to deny all resources. %v`, denyerr)
 	}
 
-	return allow, deny
+	return allow, deny, allowerr, denyerr
```

Comment:
Why are we returning this error when we don't take any further action based on it? https://github.com/stolostron/search-collector/blob/f6ba1a73d94e169c1eeea99eca1df37d3eee339f/pkg/informer/supportedResources.go#L119

Reply:
We return these errors so that we can validate the errors are the same in the test case.

Reply:
(they are only used for the unit test)